doc/nrf/links.txt

```diff
@@ -115,10 +115,10 @@
 .. _`Getting started guide for nRF Asset Tracker for AWS`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.6.x/docs/aws/GettingStarted/Index.html
 .. _`Getting started guide for nRF Asset Tracker for Azure`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.6.x/docs/azure/GettingStarted/Index.html
 
-.. _`nRF Connect for Visual Studio Code`: https://nordicplayground.github.io/vscode-nrf-connect/
-.. _`Creating an application`: https://nordicplayground.github.io/vscode-nrf-connect/#creating-an-application
-.. _`Building an application`: https://nordicplayground.github.io/vscode-nrf-connect/#building-an-application
-.. _`Testing an application`: https://nordicplayground.github.io/vscode-nrf-connect/#testing-an-application
+.. _`nRF Connect for Visual Studio Code`: https://nrfconnect.github.io/vscode-nrf-connect/
```

Comment:
This seems to be the only link that works for me, the others lead to a 404 error. But maybe that's because those links are not ready yet?

Reply:
Ah it was the / that was breaking the link. I have updated the links without them. Can you confirm these work now?

Reply:
Yes, they are all working now!